### ISSUE TITLE
[userProgress] PHP Error when using the Style

### ISSUE THREAD

**Issue by Kodzhabashev** [humdek-unibe-ch/sh-selfhelp#192]
In GitLab by @moiri on Sep 13, 2019, 16:31

The style `userProgress` produces the following PHP error:

```
Fatal error: Uncaught Error: Call to undefined method BaseStyleModel::get_user_progress() in /home/moiri/Documents/slp-selfhelp/git-slp-selfhelp/server/component/style/userProgress/UserProgressView.php:23

Stack trace:
#0 /home/moiri/Documents/slp-selfhelp/git-slp-selfhelp/server/component/style/BaseStyleComponent.php(46): UserProgressView->__construct(Object(BaseStyleModel))
#1 /home/moiri/Documents/slp-selfhelp/git-slp-selfhelp/server/component/style/StyleComponent.php(71): BaseStyleComponent->__construct('userProgress', Array, Array)
#2 /home/moiri/Documents/slp-selfhelp/git-slp-selfhelp/server/component/style/StyleModel.php(88): StyleComponent->__construct(Object(Services), 62, Array)
#3 /home/moiri/Documents/slp-selfhelp/git-slp-selfhelp/server/component/style/StyleComponent.php(68): StyleModel->__construct(Object(Services), 19, Array)
#4 /home/moiri/Documents/slp-selfhelp/git-slp-selfhelp/server/component/cms/CmsView.php(180): StyleComponent->__construct(Object(Services), 19)
#5 /home/moiri/Documents/slp-selfhelp/git in /home/moiri/Documents/slp-selfhelp/git-slp-selfhelp/server/component/style/userProgress/UserProgressView.php on line 23
```

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Sep 18, 2019, 12:49

closed via commit 57b99a127ddb1ed388db4a432e0ec7783c2df5db

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Sep 18, 2019, 14:15

mentioned in commit 57b99a127ddb1ed388db4a432e0ec7783c2df5db
